### DISCUSSION TITLE
using YAML/JSON/biblatex with ACM CHI extension?

### DISCUSSION THREAD

**Discussion by reagle** [quarto-dev/quarto-cli#2075] [Q&A]
Hello, I'm a Quarto newbie and am trying to create a submission for CHI'23. I use pandoc for my own materials and have bibliographies in CSL YAML & JSON. However, the [ACM extension](https://quarto.org/docs/journals/formats.html) seems to require bibtex files...(?) I can use pandoc to convert from YAML to bibtex, but it is very lossy (e.g., dropping `url`). My bibliography tool can also generate biblatex, but quarto/xetex complains I'm missing the `year` field -- I'm using the `date` field. Any suggestions for:

- being able to use CSL YAML/JSON "natively"?
- converting from CSL/biblatex to bibtex?

Thanks!

**Comment by cscheid** [comment]
@reagle Thanks for the interest! I know our CHI format is sub-optimal, and I'd love help you and create a top-notch CHI format. Would you mind reaching out over email to me at carlos.scheidegger@rstudio.com?

**Reply by reagle** [reply]
Hi, just wondering if there's been any progress on this?

**Reply by mcanouil** [reply]
As far as I can see there were some updates in the template, did you try the latest version of it?
See <https://github.com/quarto-journals/acm>.
